TimberAPIExample/ExampleAssets/Specifications/v0.2.1.0/bonustypes/BonusTypeSpecification.Assembly.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
200 files of specification as they are in the game? :D I'm just gonna assume they are all fine

Reply:
Me too!